src/components/MostPopular/ProductItem/ProductItem.js

```diff
@@ -7,21 +7,28 @@ import './ProductItem.scss';
 const CN = 'product';
 const ProductItem = ({ product }) => {
   const [isHovered, setHovered] = useState(false);
-  const { image, label, title, price, sizes } = product;
+  const {
+    image,
+    label,
+    title,
+    price,
+    sizes
+  } = product;
 
-  const renderView = (isHovered) => (
-    <Fragment>
+  const renderView = () => (
+    <>
       <div className={cx(`${CN}__img-wrapper`, { [`${CN}__img-wrapper--hovered`]: isHovered })}>
         {!isHovered && (
           <>
```

Comment:
You don't need this fragment, you can use [] to render several divs, smth like this: https://til.hashrocket.com/posts/oqjntperif-render-an-array-of-elements-with-react-16